XamarinCommunityToolkitSample/ViewModels/Behaviors/TagBehaviorViewModel.cs

```diff
@@ -0,0 +1,19 @@
+﻿
+using System.Windows.Input;
+using Xamarin.Forms;
+
+namespace Microsoft.Toolkit.Xamarin.Forms.Sample.ViewModels.Behaviors
+{
+    public class TagBehaviorViewModel : BaseViewModel
+    {
+        string text = "Hello, here is a list of hashtags and other types of tags: #DotNetMAUI #Community #OpenSource @xamarinhq @dotnet @microsoft #MicrosoftToolkitXamarinForms #Xamarin #Monkeys #XamarinForms @planetxamarin #ILoveXamarin #CSharp #Behaviors";
+
+        public string Text
+        {
+            get => text;
+            set => Set(ref text, value);
+        }
+
+        public ICommand TagTappedCommand => new Command<string>(async (s) => await Application.Current.MainPage.DisplayAlert("Tag Tapped:", s, "Ok"));
```

Comment:
```csharp
ICommand tagTappedCommand;

public ICommand TagTappedCommand => tagTappedCommand ??= new Command(...)
```

Reply:
Did this, but I'm not sure what the reasoning for this is?

Reply:
A small memory optimization.

**#1**
```public ICommand TagTappedCommand => new Command()```

**#2**
ICommand tagTappedCommand;
```public ICommand TagTappedCommand => tagTappedCommand ??= new Command()```

**Case**

```csharp
for (var i = 0; i < 1000; ++i) {
    TagTappedCommand.Execute(null);
}
```
In case **#1** a new property will be created each time when it is accessed (1000 times)
In case **#2** the property will be created only once.